Demo/Demo/ViewModels/BaseViewModel.swift

```diff
@@ -120,12 +120,11 @@ class BaseViewModel: ObservableObject, PayPalWebCheckoutDelegate, CardDelegate {
         return Card(number: cleanedCardText, expirationMonth: expirationMonth, expirationYear: expirationYear, securityCode: cvv)
     }
 
-    func checkoutWithCard(_ card: Card, orderID: String, context: ASWebAuthenticationPresentationContextProviding) async {
-        let config = CoreConfig(
-            clientID: DemoSettings.clientID,
-            environment: DemoSettings.environment.paypalSDKEnvironment,
-            secret: DemoSettings.secret
-        )
+    func checkoutWithCard(_ card: Card, orderID: String,
+            context: ASWebAuthenticationPresentationContextProviding) async {
```

Comment:
I think we can keep this as 1 line unless we are getting a linter warning for line length. If we are this is the format we would follow:

```suggestion
    func checkoutWithCard(
        _ card: Card, 
        orderID: String,
        context: ASWebAuthenticationPresentationContextProviding
    ) async {
```